src/auth/Crypto.cc

```diff
@@ -348,24 +360,695 @@ int CryptoAES::validate_secret(const bufferptr& secret)
   return 0;
 }
 
-CryptoKeyHandler *CryptoAES::get_key_handler(const bufferptr& secret,
-					     string& error)
+CryptoKeyHandler *CryptoAES::get_key_handler(
+    const bufferptr& secret,
+		string& error)
```

Comment:
style nit: can we stick with the existing formatting/indentation throughout? `void foo(arg1, arg2, arg3);`